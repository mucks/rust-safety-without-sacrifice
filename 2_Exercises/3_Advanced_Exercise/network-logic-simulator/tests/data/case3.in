# DAG job scheduling with resources
JOB compile dur=120 req=2:512
JOB link dur=60 req=1:256 DEPENDS compile
JOB pack dur=30 req=1:256 DEPENDS link
JOB test dur=90 req=2:256 DEPENDS compile
SCHEDULE policy=shortest
