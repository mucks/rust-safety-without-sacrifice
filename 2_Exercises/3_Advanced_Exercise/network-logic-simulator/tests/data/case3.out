SCHEDULED:
t=0..60    link      cpu=1 mem=256 deps_satisfied=false
t=0..30    pack      cpu=1 mem=256 deps_satisfied=false
t=0..90    test      cpu=2 mem=256 deps_satisfied=false
t=0..120   compile   cpu=2 mem=512 deps_satisfied=true
ORDER: link, pack, test, compile
TOTAL_DURATION_MS: 120
